src/components/editMeasure/testCases/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/ElementEditorActionCenter.tsx

```diff
@@ -3,92 +3,133 @@ import { SpeedDial, SpeedDialAction } from "@mui/material";
 import DeleteOutlinedIcon from "@mui/icons-material/DeleteOutlined";
 import ContentCopyIcon from "@mui/icons-material/ContentCopy";
 import AddCircleOutlineIcon from "@mui/icons-material/AddCircleOutline";
-import { blue, red } from "@mui/material/colors";
 
 interface PropTypes {
   //coming soon
+  //  ...and it's HERE!  Thanks to GAK & MAT-8338
+  rootDefinition: any;
+  numElements: number;
+}
+interface Action {
+  name: string;
+  icon: JSX.Element;
 }
-
 const ElementEditorActionCenter = (props: PropTypes) => {
   const [open, setOpen] = useState(false);
+  const [numElements, setNumElements] = useState<number>(props.numElements);
```

Comment:
Do we need a state for this ?

Reply:
I'm not sure..  I was having troubles getting the form elements to refresh, so I moved that into state to help

Reply:
Lets merge your PR there are going to be tons between your work, mine and Ethan's